packages/libs/error-stack/CHANGELOG.md

```diff
@@ -11,13 +11,16 @@ All notable changes to `error-stack` will be documented in this file.
 ### Breaking Changes
 
 - Remove all previously deprecated methods ([#1485](https://github.com/hashintel/hash/pull/1485))
+- Remove `pretty-print` feature ([#1800](https://github.com/hashintel/hash/pull/1800))
 
 ### Features
 
-- Add serializing support using [`serde`](https://serde.rs) ([#1290](https://github.com/hashintel/hash/pull/1290))
+- Add initial serializing support using [`serde`](https://serde.rs) ([#1290](https://github.com/hashintel/hash/pull/1290))
 - Support `Debug` hooks on `no-std` platforms via the `hooks` feature ([#1556](https://github.com/hashintel/hash/pull/1556))
 - Support converting `Report` into [`Error`](https://doc.rust-lang.org/core/error/trait.Error.html) via `Report::as_error` and `Report::into_error` ([#1749](https://github.com/hashintel/hash/pull/1749))
 - Support converting `Report` into `Box<dyn Error>` via the `From` trait ([#1749](https://github.com/hashintel/hash/pull/1749))
+- Programmatic selection of color mode and charset used for `Debug` output ([#1800](https://github.com/hashintel/hash/pull/1800))
+- New feature `color`, supersedes `pretty-print` feature ([#1800](https://github.com/hashintel/hash/pull/1800))
```

Comment:
Just as a reminder, that we have to tweak this entry when removing the feature again 🙂